src/tools/extractor_common/CascHandles.cpp

```diff
@@ -190,6 +190,31 @@ CASC::Storage* CASC::Storage::Open(boost::filesystem::path const& path, uint32 l
     return storage;
 }
 
+CASC::Storage* CASC::Storage::OpenRemote(uint32 localeMask, char const* product)
+{
+    HANDLE handle = nullptr;
+    boost::filesystem::path cachDir = boost::filesystem::current_path() / "Cache";
+
+    std::string cacheArgument = std::string(cachDir.string() + ":" + product + ":eu");
```

Comment:
I also would make some sort of argument to change the locale here

Reply:
do you mean the region eu/us?

Reply:
Yes